manifests/i/Instatus/Out/1.0.7/Instatus.Out.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Instatus.Out
+PackageVersion: 1.0.7
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/instatushq/out/releases/download/v1.0.7/Instatus-Out-Setup-1.0.7.exe
+  InstallerSha256: e28773bc9c2465f9061de37bc59f857fd9bde9c0f73a1784b24dc2b128f8fefd

```

```suggestion
  InstallerSha256: e28773bc9c2465f9061de37bc59f857fd9bde9c0f73a1784b24dc2b128f8fefd
  Scope: user
```
`C:\Users\%USERNAME%\AppData\Local\Programs\Instatus Out`